aiida_nanotech_empa/workflows/gaussian/delta_scf_workchain.py

```diff
@@ -0,0 +1,244 @@
+        parameters = Dict(
+            dict={
+                'link0_parameters': self.ctx.link0.copy(),
+                'functional': functional,
+                'basis_set': self.inputs.basis_set.value,
+                'charge': 1,
+                'multiplicity': pos_mult,
+                'route_parameters': {
+                    'scf': {
+                        'maxcycle': 128,
+                        'conver': 7
+                    },
+                    'nosymm': None,
+                    'sp': None,
+                },
+            })
+
+        builder = GaussianBaseWorkChain.get_builder()
+
+        if pos_mult == 1:
+            # For open-shell singlet, mix homo & lumo
+            parameters['route_parameters']['guess'] = "mix"
+
+        builder.gaussian.parameters = parameters
+        self.setup_common_builder_params(builder)
+        submitted_node = self.submit(builder)
+        self.to_context(pos=submitted_node)
+
+        # --------------------------------------------------
+        self.report("Submitting ANION SCF")
+        # --------------------------------------------------
+        # For the ANION, the added electron could go opposite or parallel
+        # if the system was already spin-polarized
+
+        if self.ctx.mult == 1:
+            self.ctx.neg_mults = [2]
+        else:
+            self.ctx.neg_mults = [self.ctx.mult - 1, self.ctx.mult + 1]
+
+        for neg_mult in self.ctx.neg_mults:
+            parameters = Dict(
+                dict={
+                    'link0_parameters': self.ctx.link0.copy(),
+                    'functional': functional,
+                    'basis_set': self.inputs.basis_set.value,
+                    'charge': -1,
+                    'multiplicity': neg_mult,
+                    'route_parameters': {
+                        'scf': {
+                            'maxcycle': 128,
+                            'conver': 7
+                        },
+                        'nosymm': None,
+                        'sp': None,
+                    },
+                })
+
+            builder = GaussianBaseWorkChain.get_builder()
+
+            if neg_mult == 1:
+                # For open-shell singlet, mix homo & lumo
+                parameters['route_parameters']['guess'] = "mix"
+
+            builder.gaussian.parameters = parameters
+            self.setup_common_builder_params(builder)
+            submitted_node = self.submit(builder)
+            label = "neg_m%d" % neg_mult
+            self.to_context(**{label: submitted_node})
+
+    def finalize(self):
+
+        if (not common.check_if_previous_calc_ok(self, self.ctx.neutral)
+                or not common.check_if_previous_calc_ok(self, self.ctx.pos)):
+            return self.exit_codes.ERROR_TERMINATION
+
+        anion_energies = []
+
+        for neg_mult in self.ctx.neg_mults:
+            label = "neg_m%d" % neg_mult
+            if not common.check_if_previous_calc_ok(self, self.ctx[label]):
```

Comment:
```suggestion
            if not common.check_if_previous_calc_ok(self, self.ctx[f"neg_m{neg_mult}"]):
```

there are many similar things, please fix them.